### DISCUSSION TITLE
The third feature , we may add a button in the main page

### DISCUSSION THREAD

**Discussion by Jasonlovedataanalyst** [CarlosPietrasanta/Tafe230130#2] [Ideas]
@ozdotcom @CarlosPietrasanta 

below are the info that I got from the teacher , just want to check whether we should add an additional button in the main repo**** 

**** below is quoted from the teacher's email 

I have added screenshots of the third feature you will introduce to the calculator program. 

You and the team will need to add one more button to the menu to include a link to this page.

 

Thanks everyone for being so accommodating. Jason has suggested he uses git at work so he will be a good addition to the group and help out where he can.

 

Please reach out if you need help or support

 

 
![third feature 01](https://user-images.githubusercontent.com/87595451/226455224-529afdcf-e251-4054-b061-3e5fd41e92f9.png)

 
![third feature02](https://user-images.githubusercontent.com/87595451/226455241-1e05fbf8-74d3-4691-b218-a76708df4405.png)

**Comment by Jasonlovedataanalyst** [comment]
for this one , I reckon I could do it in the feature branch for the Trip Calculator , and then merge into the master later on , so that the button will be shown there. @ozdotcom @CarlosPietrasanta

**Comment by CarlosPietrasanta** [comment]
@Jasonlovedataanalyst Thanks for the heads up Jason. Fortunately I got a copy of the email as well and added the button to the first version of the main menu. I'll attach a screenshot so you can have a look at it. Please feel free to suggest any recommendations.

![Screenshot 2023-03-21 123846](https://user-images.githubusercontent.com/127190272/226503037-846d74da-1d28-4460-bd4b-2e664f6f03e2.png)

**Comment by ozdotcom** [comment]
Awesome work Carlos and Jason.

Can't add much to this discussion, you guys are on top of it.
